test/ops/align.jl

```diff
@@ -168,6 +168,18 @@ end
     @test _eval(count_knots(n_boolean)) == _expected_count_knots(b_boolean)
 end
 
+@testset "skip" begin
+    @test _eval(skip(n1, 1)) == b1[2:end]
+    @test _eval(skip(n1, 3)) == b1[4:end]
+    @test _eval(skip(n1, 10)) == _eval(empty_node(Float64))
+    @test skip(constant(5), 1) === empty_node(Int)
+    @test skip(constant(5), 0) === constant(5)
+    @test skip(n1, 1) === skip(n1, 1)
+    @test skip(n1, 0) === n1
+    @test_broken empty_node(Float64) === skip(empty_node(Float64), 3)
```

Comment:
(Reverse order of equality check? Currently in Yoda form. Using @test_broken is a good idea, I keep forgetting that this exists!)